src/industry_gui.cpp

```diff
@@ -901,6 +893,9 @@ class IndustryViewWindow : public Window
 		bool first = true;
 		bool has_accept = false;
 
+		/* Use all the available space past the rect, so that we can enlarge the window if needed. */
+		ir.bottom = INT_MAX;
+
```

Comment:
Is the deja-vu/conflict with #13643 intentional.

Reply:
Yeah, this is on top of #13643 as it touches the same lines.